_posts/2018-03-01-Rust-1.24.1.md

```diff
@@ -156,9 +157,10 @@ Consider the code above, and imagine that `lua_newtable` here could call
 3. The initial `lua_pcall` catches the `longjmp` with the `setjmp` it called earlier.
 4. Everyone is happy.
 
-However, `lua_newtable` is an `extern fn` to Rust. So, with the chances in
-1.24.1, it sets up a panic handler that will cause an abort. In other words,
-the code sorta looks something like this psuedo code now:
+However, the implementation of `protect_lua_call` converts our closure to an
+`extern fn`, since that's what Lua needs. So, with the changes in 1.24.0, it
+sets up a panic handler that will cause an abort. In other words, the code
+sorta looks something like this pseudo code now:
```

Comment:
"Sorta" is unnecessary here. (Side note: what's our desired level of formality for release notes? "Sorta" is sorta informal.)

Reply:
Our voice is not totally formal. rust is a friendly language :)